src/Common/Models/PolicyModel.cs

```diff
@@ -11,11 +11,17 @@ public class PolicyModel : BaseModel
     public override string Name { get; set; }
 
     [JsonPropertyName("Timestamp")]
-    public DateTime Timestamp { get; set; }
+    public DateTime? Timestamp { get; set; }
 
     [JsonPropertyName("IsActive")]
     public bool? IsActive { get; set; }
 
     [JsonPropertyName("Description")]
     public string Description { get; set; }
+
+    [JsonPropertyName("CannotCoexistFamily")]
+    public int CannotCoexistFamily { get; set; }
```

Comment:
Is it actually a good name? Maybe something like `IsExclusiveWithinType` will be better?
Also, as a side note. Usually when properties or methods are defied and they return a boolean value, hteir name indicates that the statement is true. It is very much helpful from the understanding point of view as well as readability.
Example would be to have `CanCoexistFamily` instead of `CannotCoexistFamily`.

Reply:
done